core/DataTable/Filter/ColumnDelete.php

```diff
@@ -160,11 +160,15 @@ protected function removeColumnsFromTable(&$table)
                 continue;
             }
             foreach ($this->columnsToRemove as $column) {
-
-                if (!array_key_exists($column, $row)) {
-                    continue;
+                if (is_array($row )) {
```

Comment:
Seems like there is a space too much here:
```suggestion
                if (is_array($row)) {
```

Reply:
changed @Findus23